pyproject.toml

```diff
@@ -23,6 +22,7 @@ classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Environment :: Console",
     "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
```

Comment:
We only need this line for backwards compatibility with pip-licenses, the rest can be PEP 639: https://github.com/raimon49/pip-licenses/blob/3fbf20f9d757d2b9298dfc97af1bd3a987124c98/piplicenses.py#L603-L612